### PR TITLE
ITEX[CPU] with AMP BF16 support 

### PR DESCRIPTION
**Pull request by asirvaiy**
Hi, Wanted to update the repo with Intel Extension for TensorFlow(ITEX) for CPU with AMP BF16. ITEX[GPU] is in progress, as it's having some issues currently.

### CODE REVIEW

File: 2D/README.md [hunk 1]
```diff
@@ -7,37 +7,89 @@ Please see our blog on the [IntelAI website](https://www.intel.ai/intel-neural-c
...
 
-3. We use [conda virtual environments](https://www.anaconda.com/distribution/#download-section) to run Python scripts. Once you download and install conda, create a new conda environment with [TensorFlow* with Intel&reg; DNNL](https://software.intel.com/en-us/articles/intel-optimization-for-tensorflow-installation-guide?page=1). Run the command: 
+## Steps to train the model 
+
+1. We use [conda virtual environments](https://www.anaconda.com/distribution/#download-section) to run Python scripts. Once you download and install conda, create a new conda environment with [TensorFlow* with Intel&reg; DNNL](https://software.intel.com/en-us/articles/intel-optimization-for-tensorflow-installation-guide?page=1). Run the command: 
 ```
 conda create -c anaconda -n decathlon pip python=3.7 tensorflow tqdm psutil jupyter matplotlib
 ```
 
 This has been tested with [TensorFlow 2.2](https://anaconda.org/anaconda/tensorflow-mkl) on Ubuntu 18.04 Linux.
```
Comment (ravi9) [inline 2D/README.md:22]: @asirvaiy Need to be updated to Tensorlfow 2.10 ?

**Review by ravi9 — CHANGES_REQUESTED** [review]

File: 2D/README.md (hunk 1 above)
Comment (asirvaiy) [reply 2D/README.md:22]: Hi @ravi9 yes, I tested with TF v2.10. So, 2.2 can be replaced. Also python version was 3.9 or 3.10

Comment (ravi9) [reply 2D/README.md:22]: Ok, could you update it please.

File: 2D/02_Inference.ipynb [hunk 2]
```diff
@@ -28,22 +28,95 @@
    "metadata": {},
```
Comment (ravi9) [inline 2D/02_Inference.ipynb:28]: When I try to view the notebook in GitHub, I get the error: Invalid Notebook. Could you check @asirvaiy ..

**Review by ravi9 — CHANGES_REQUESTED** [review]

File: 2D/02_Inference.ipynb (hunk 2 above)
Comment (asirvaiy) [reply 2D/02_Inference.ipynb:28]: Hi @ravi9 Is this issue only for this notebook or all the notebooks(updated)?

Comment (asirvaiy) [reply 2D/02_Inference.ipynb:28]: The issue is for inference-solution.ipynb notebook due to missing ending syntax. Fixing it.

Comment (asirvaiy) [reply 2D/02_Inference.ipynb:28]: Now inference-solution.ipynb should open @ravi9 please check.

**Pull request merged**